tasks/components/user_input/Textarea.tsx

```diff
@@ -0,0 +1,40 @@
+import { css, tw } from '@helpwave/common/twind'
+import type { TextareaHTMLAttributes } from 'react'
+
+type TextareaProps = {
+  headline?: string,
+  placeholder?: string,
+  message?: string,
+  id?: string,
+  onChange: (text: string) => void,
+  props?: TextareaHTMLAttributes<HTMLTextAreaElement>
+}
+
+const globalStyles = css`
+  /* onfocus textarea border color */
+  .textarea-wrapper:focus-within {
+    @apply border-hw-primary-600;
+  }
+
+  /* style of the scrollbar of the textarea */
+  textarea::-webkit-scrollbar {
+    @apply w-1;
+  }
+
+  textarea::-webkit-scrollbar-thumb {
+    @apply rounded-lg bg-hw-primary-600;
+  }
```

Comment:
The Scrollbar styling should be applied globally and not just for this component's scrollbar.

You can just omit it here, and we will open an additional issue for that

Reply:
Sound good to me

Reply:
++ #163